packages/appstore/src/components/wallet-cards-carousel/cards-slider.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { ProgressBarOnboarding, WalletCard } from '@deriv/components';
+import { TWalletAccount } from 'Types';
+import { formatMoney } from '@deriv/shared';
+import Slider from 'react-slick';
+
+type TProps = {
+    readonly items: TWalletAccount[];
+    setActivePage: React.Dispatch<React.SetStateAction<number>>;
+};
+
+const CardsSlider = React.memo(({ items, setActivePage }: TProps) => {
+    const slider_ref = React.useRef<Slider | null>(null);
+
+    const handlerGoTo = (slideNumber: number) => {
+        setActivePage(slideNumber - 1);
+        slider_ref?.current?.slickGoTo(slideNumber - 1);
+    };
+
+    const settings = {
+        dots: true,
+        infinite: false,
+        centerMode: true,
+        centerPadding: '0',
+        slidesToShow: 1,
+        slidesToScroll: 1,
+        swipeToSlide: true,
+        variableWidth: true,
+        arrows: false,
+        speed: 500,
+        touchThreshold: 10,
+        dotsClass: 'wallet-cards-carousel__pagination',
+        afterChange: (currentSlide: number) => {
+            setActivePage(currentSlide);
+        },
+        appendDots: (dots: React.ReactElement[]) => {
+            const active = dots.findIndex(dot => dot.props.className === 'slick-active') || 0;
+            const steps = dots.map((_, idx) => idx.toString());
+
+            return (
+                <div>
+                    <ProgressBarOnboarding
+                        step={active + 1}
+                        amount_of_steps={steps}
+                        is_transition={true}
+                        setStep={handlerGoTo}
+                    />
+                </div>
+            );
+        },
+    };
+
+    return (
+        <Slider {...settings} ref={slider_ref}>
+            {items.map((item: TWalletAccount) => (
```

Comment:
@sergei-deriv we can make this component a blind one and just pass items to it to render. In this case you need to move it to the components package and reuse it here and pass wallet items to it.

Reply:
@mahdiyeh-deriv Honestly, I don't understand what I have to move to components package. `Slider` component is a component from 3rd party library and I pass items as `children` for it. Also I have settings for `Slider` here, so for new component, for example, `TradingAccountsSlider` I will use `Slider` again, will have own settings and pass other items as `children`. Let me know what I need to move because I don't fully understand.

Reply:
@sergei-deriv What I mean is that I want to make this component a dummy one which gets items and show them in the slider. No matter what are the items and what is the details in it